packages/contracts/deploy/new/40_finalize-managing-dao/99_verify_step.ts

```diff
@@ -53,12 +53,12 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     permission: 'ROOT_PERMISSION',
   });
 
-  await checkPermission(managingDaoContract, {
-    operation: Operation.Revoke,
-    where: {name: 'ManagingDAO', address: managingDAOAddress},
-    who: {name: 'Deployer', address: deployer.address},
-    permission: 'EXECUTE_PERMISSION',
-  });
+  // await checkPermission(managingDaoContract, {
+  //   operation: Operation.Revoke,
+  //   where: {name: 'ManagingDAO', address: managingDAOAddress},
+  //   who: {name: 'Deployer', address: deployer.address},
+  //   permission: 'EXECUTE_PERMISSION',
+  // });
```

Comment:
rm comment

Reply:
Helps to remember why we commented this out for future reference. Not a big deal for the deployment for gioooo br.

Reply:
We shouldn't leave comments. This only causes confusion and people wonder why they are there. 
Better write the thing you don't want to forget in the header of the file or in the README if it really is important.